docs/development/runtime-running.md

```diff
@@ -0,0 +1,69 @@
+---
+description: Running the runtime
+---
+
+# Runtime Execution
```

Comment:
I think that we already covered the compilation and chain spec in the previous pages. Here we should describe the actual process of running the runtime in a node in more details